artifactory_cleanup/artifactorycleanup.py

```diff
@@ -69,6 +69,9 @@ def _destroy_or_verbose(self):
             print("Verbose MODE")
 
     def main(self):
+        if self._artifactory_server.endswith("/"):
```

Comment:
We can `rstrip`, without `if statement. Rstrip won't do anything if string ends no with  "/".